fetch-in-react/my-app/src/User.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import UserCard from './UserCard';
+
+export default function User({ userId, onCancel }) {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [user, setUser] = useState();
+
+  /* your code here (hint: useEffect) */
+
```

Comment:
I don't see any new code here?

Reply:
Hello Robert, sorry that I missed the second part of the assignment.I have revised and repushed the solution.